bucket/throttlestop.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "8.70.6",
+    "description": "Monitor and adjust CPU throttling on laptop computers.",
+    "homepage": "https://www.techpowerup.com/download/techpowerup-throttlestop",
```

Comment:
Why do you change something, which is not broken?
For hashes it is intentional to follow that redirection, since they use load balancers and it is not nginx based.

![image](https://user-images.githubusercontent.com/13260377/73589624-d4151e80-44d8-11ea-9322-e844dc56cce9.png)

Reply:
> For hashes it is intentional to follow that redirection, since they use load balancers and it is not nginx based.

Didn't think about that. I just wanted to keep the URL the same as `autoupdate` section. Thanks for the details.